packages/api-v2/src/hooks/useSortedMT5Accounts.ts

```diff
@@ -71,12 +82,17 @@ const useSortedMT5Accounts = (regulation?: string) => {
 
     // Sort the data by market_type to make sure the order is 'synthetic', 'financial', 'all'
     const sorted_data = useMemo(() => {
-        const market_type_order = ['synthetic', 'financial', 'all'];
+        const market_type_order = ['synthetic', 'financial', 'swap_free', 'zero_spread'];
 
         if (!filtered_data) return;
 
         const sorted_data = market_type_order.reduce((acc, market_type) => {
-            const accounts = filtered_data.filter(account => account.market_type === market_type);
+            const accounts = filtered_data.filter(account => {
+                if (account.market_type === 'all') {
+                    return account.product === market_type;
```

Comment:
this part of the code confuses me too... why are you setting `account.product === market_type`

Reply:
Changed the confusing variable name...